src/geos.h

```diff
@@ -4,7 +4,9 @@
 #include <Python.h>
 
 /* To avoid accidental use of non reentrant GEOS API. */
+#ifndef GEOS_USE_ONLY_R_API
 #define GEOS_USE_ONLY_R_API
+#endif
 
 // wrap geos.h import to silence geos gcc warnings
 #pragma GCC diagnostic push
```

Comment:
Do the following "#pragma GCC" statements make any difference to the outputs? From what I've seen, the "geos_c.h:160:1: warning: function declaration isn’t a prototype" messages persist on the Linux builds, while the Windows builds show warnings: "geos.h(12): warning C4068: unknown pragma 'GCC'"

Reply:
Yes - these were what originally what silenced the warnings via #115 

The reason the warnings are back is because `geos.h` is imported into the Cython `_geos.pxd` file, but in a way that doesn't allow us to wrap that import like this (as far as I could tell).